dev/Deployment/DeploymentManager.cpp

```diff
@@ -105,12 +105,12 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
             if (package.versionType == PackageVersionType::Versioned)
             {
                 // Prefix + SubTypeName + VersionIdentifier + Suffix
```

Comment:
Please consider updating the comments here and above to detail what GetStatus is looking for in terms of verifying the Singleton and Main packages etc.